src/ol_openedx_git_auto_export/README.md

```diff
@@ -0,0 +1,53 @@
+#### Steps to install on edX platform:
+
+
+You can install this plugin into any Open edX instance by using any of the following methods:
+
+
+**Option 1: Install from PyPI**
+
+.. code-block::
+
+    # If running devstack in docker, first open a shell in LMS (make lms-shell)
+
+    pip install ol-openedx-git-auto-export
+
+
+**Option 2: Build the package locally and install it**
+
+Follow these steps in a terminal on your machine:
+
+1. Navigate to ``open-edx-plugins`` directory
+2. If you haven't done so already, run ``./pants build``
+3. Run ``./pants package ::``. This will create a "dist" directory inside "open-edx-plugins" directory with ".whl" & ".tar.gz" format packages for all the "ol_openedx_*" plugins in "open-edx-plugins/src")
+4. Move/copy any of the ".whl" or ".tar.gz" files for this plugin that were generated in the above step to the machine/container running Open edX (NOTE: If running devstack via Docker, you can use ``docker cp`` to copy these files into your LMS container)
+5. Run a shell in the machine/container running Open edX, and install this plugin using pip
+
+
+- open cms.env and lms.env or private.py and set FEATURES flags
+
+  ```
+  "FEATURES": {
+    "ENABLE_EXPORT_GIT": true,
+    "ENABLE_GIT_AUTO_EXPORT": true
+  }
+  ```
```

Comment:
In addition to the feature flag, there is one more thing that needs to be set for clarity and that confused. Looking at the [history of commits](https://github.com/umarmughal824/test_edx_course/commit/9f55ac9bfadd9876d18c069e58ed03a11522ef48) on your fork it seems to me that it confused you too with author/commit name as `STUDIO_EXPORT_TO_GIT`. **This means it's worth mentioning that you need to set thses in order for your commits to be commissioned properly**.

[Reference from platform](https://github.com/mitodl/edx-platform/blob/master/cms/envs/common.py#L1074-L1077)

```
GIT_EXPORT_DEFAULT_IDENT = {
    'name': 'STUDIO_EXPORT_TO_GIT',
    'email': 'STUDIO_EXPORT_TO_GIT@example.com'
}
```